packages/page-template-modal/src/components/template-selector-preview.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import BlockIframePreview from './block-iframe-preview';
+
+const TemplateSelectorPreview = ( { blocks = [], viewportWidth, title } ) => {
+	const noBlocks = ! blocks.length;
+	return (
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
+		<div className={ `template-selector-preview ${ noBlocks ? 'not-selected' : '' }` }>
+			{ noBlocks && (
+				<div className="editor-styles-wrapper">
+					<div className="template-selector-preview__empty-state">
+						{ __( 'Select a layout to preview.', 'full-site-editing' ) }
```

Comment:
Do we have to use `__i18n_text_domain__` here (like we do in [other packages](https://github.com/Automattic/wp-calypso/blob/trunk/packages/plans-grid/src/plans-details/index.tsx#L55)) so that we can use the right domain depending on context?

```
{ __( 'Select a layout to preview.', __i18n_text_domain__ ) }
```

cc @p-jackson Who might be able to answer as well

Reply:
Yup, everywhere that used to be `__( '*', 'full-site-editing' )` now needs to be `__( '*', __i18n_text_domain__ )`.
Might also need a couple TS tweaks to make the compiler happy too, let me rustle those up ...

Reply:
Never mind, I forgot it was JS, not TS. When we do starting writing some TS we'll want to add a declaration for the `__i18n_text_domain` global, like this one here: https://github.com/Automattic/wp-calypso/blob/9555b50c62e047790e185775c54c302ffe066151/packages/onboarding/src/types.d.ts#L1

And when we start writing tests we'll need to make sure our jest config inherits from the shared package one here: https://github.com/Automattic/wp-calypso/blob/9555b50c62e047790e185775c54c302ffe066151/packages/onboarding/jest.config.js#L2 so that tests don't fail due to the missing global.

Reply:
We also don't have to do it in this PR if we add an item to the board :)

Reply:
thanks I've updated to use `__i18n_text_domain__`